Controller/src/controller/rpc.go

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package controller
+
+var cert []byte
+
+func initServer() {
+	err := makeCert()
+	if err != nil {
+		return err
+	}
+	tls, err := credentials.NewServerTLSFromFile("cert.pem", "key.pem")
+	if err != nil {
+		return err
+	}
+	srv := grpc.NewServer(grpc.Creds(tls))
+	lis, err := net.Listen("tcp", "crdhost.c.gifted-cooler-279818.internal:50001")
```

Comment:
Don't hardcode this address, it should be a flag

Reply:
Addressed in 85540dd